libqtile/backend/wayland/core.py

```diff
@@ -587,8 +647,20 @@ def _on_cursor_button(self, _listener: Listener, event: pointer.PointerButtonEve
             handled = self._process_cursor_button(button, pressed)
 
         if not handled:
+            if self._pressed_button_count == 1 and found:
+                assert self._implicit_grab is None
+                win, surface, sx, sy = found
+                if surface:
+                    self._create_implicit_grab(event.time_msec, surface, sx, sy)
             self.seat.pointer_notify_button(event.time_msec, event.button, event.button_state)
 
+    def _implicit_grab_motion(self, time: int):
```

Comment:
This may need a `-> None` return annotation

Reply:
Done.  I'll push another commit shortly.